libs/aws/odc/aws/queue.py

```diff
@@ -3,6 +3,58 @@
 from typing import Mapping, Any, Iterable, Optional
 
 
+def redrive_queue(
+    queue_name: str,
+    to_queue_name: Optional[str] = None,
+    limit: Optional[int] = None,
+    dryrun: Optional[bool] = False,
```

Comment:
this one does not need to be optional, since default value is also `bool`. `Optional[some_type]` means can be one of `None|value of some_type` it has nothing to do with parameter having a default value and hence "being optional".

Reply:
Aha, I see. Thanks!